### PR TITLE
Add ability to create and test rpm creation using tito - actions and commands build-origin-rpm-test

### PR DESCRIPTION
**Pull request by arilivigni**
Add ability to create and test rpm creation using tito
- actions and commands build-origin-rpm-test

### CODE REVIEW

**Comment by arilivigni** [comment]
@detiber @brenton Care to take a look and is there anyone I should add to the review?

**Comment by detiber** [comment]
At a quick glance, I don't see any issues.

**Comment by brenton** [comment]
This looks good to me for the first task on our card.  Do you want to submit several PRs our would you like to keep updating this one?

**Comment by arilivigni** [comment]
I'll keep this one going it seems easier unless there are any objections.  Thanks for the review!

**Comment by brenton** [comment]
sounds good to me.

**Comment by arilivigni** [comment]
@detiber @brenton This has all the actions we want to build RPMs, Create a Repo, Load into base Docker image

File: lib/vagrant-openshift/action/build_origin_images.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          sudo(env[:machine], "mkdir -p #{Constants.build_dir}")
+          sudo(env[:machine], "mkdir -p #{Constants.build_dir + "builder"} && chown -R #{ssh_user}:#{ssh_user} #{Constants.build_dir}")
+
+          sudo(env[:machine], %{
+set -e
+
+pushd #{Constants.build_dir}origin/images/base
+sudo cp -r #{Constants.build_dir}x86_64 .
+sudo cp /etc/yum.repos.d/origin_local.repo .
+docker build -t openshift/origin-base -f Dockerfile.rpm .
```
Comment (brenton) [inline lib/vagrant-openshift/action/build_origin_images.rb]: I see this is building the base image.  What command would someone run to build all the other `Dockerfile.rpm`s?

Comment (arilivigni) [inline lib/vagrant-openshift/action/build_origin_images.rb]: The should be addressed with the latest Commit to this PR.  I added an option to specify images

**Comment by brenton** [comment]
I think this is good enough for now.  I think eventually what we'll do in addition to being able to pass in a list of images is just have a single method that builds all the known images in the right order.

**Pull request merged**